src/pages/Explore/explore.css

```diff
@@ -3,7 +3,8 @@
 .playlists-main .container,
 .playlists-videos-list-main .container,
 .video-main .container,
-.playlists-videos-list-main .container {
+.playlists-videos-list-main .container,
+.history-main .container {
     height: 100%;
```

Comment:
Please use a consistent unit. If using rem then pls try to be consistent with it.

Reply:
I wanted the child container to occupy the entire space that's available relative to the parent. Hence 100%.